osp-deploy/cinder/README.md

```diff
@@ -40,54 +39,51 @@ parameter_defaults:
 
 For full detailed instruction of all options please refer to [XtremIO Backend Configuration](https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/dell-emc-xtremio-driver.html#configuration-options).
 
-**iSCSI Environment sample**
+**Environment sample**
 
-With a director deployment, XtremIO iSCSI backend can be deployed using the integrated heat environment file. This file is located in the following path of the Undercloud node:
-/usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-xtremio-iscsi-config.yaml
+With a director deployment, XtremIO backend can be deployed using the integrated heat environment file. This file is located in the following path of the Undercloud node:
+/usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-xtremio-config.yaml
 
 Copy this file to a local path where you can edit and invoke it later. For example, to copy it to ~/templates/:
 
 ```bash
-$ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-xtremio-iscsi-config.yaml ~/templates/
+$ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-xtremio-config.yaml ~/templates/
 ```
-Afterwards, open the copy (~/templates/cinder-dellemc-xtremio-iscsi-config.yaml) and edit it as you see fit. The following shows a sample content of the file. The files will list optional params that the user can choose to override if they don't like the default value.
+Afterwards, open the copy (~/templates/cinder-dellemc-xtremio-config.yaml) and edit it as you see fit. The following shows a sample content of the file. The files will list optional params that the user can choose to override if they don't like the default value.
+
+```CinderXtremioStorageProtocol``` default value is iSCSI. To configure FC backend add ```CinderXtremioStorageProtocol: 'FC'```.
 
 Note that the ```resource_registry``` entry in the heat environment file must be an absolute path when you make a copy.
 
 ```yaml
 # A Heat environment file which can be used to enable a
-# Cinder Dell EMC XTREMIO iSCSI backend, configured via puppet
+# Cinder Dell EMC Xtremio backend, configured via puppet
```

Comment:
do we have XtremIO or Xtremio there?

Reply:
the same for Puppet?

Reply:
Xtremio. This is the exact copy of the file in tripleo-heat-templates repo.

Reply:
looks clean. Merging.